SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -443,4 +443,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 		*/
 	UPROPERTY(EditAnywhere, Config, Category = "Replication", meta = (DisplayName = "Enable Initial Only Replication Condition"))
 	bool bEnableInitialOnlyReplicationCondition;
+
+	/*
+	 * Enables writing of ActorSetMember and ActorGroupMember components to load balancing entities
+	 */
+	UPROPERTY(EditAnywhere, Config, Category = "Replication")
+	bool bEnableStrategyLoadBalancingComponents;
```

Comment:
I have also added bRunStrategyWorker in the ability to start strategy worker perhaps we only need one of these going forward?

Reply:
I think I can get behind that - there's no reason to have two flags that pretty much must be both "on" or "off" for any effect to be visible. @samiwh @ImprobableNic what do you think?
If we do decide on this, we'd better give it a name that encompasses this, maybe something like `bUseStrategyWorker`?

Reply:
bUseStrategyWorker is probably fine provided it's either in the experimental category or unlisted.